### ISSUE TITLE
Add accessibility definitions to PAC

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1348]
Add definitions to PAC. 

@lukefretwell @aschmoe do you think this language should be on the PAC homepage and/or each report?

**Passed**
Issues checked by the ProudCity Accessibility Checker (PAC) that pass WCAG level AA accessibility standards. 
**Errors**
Issues detected by PAC that must be fixed. 
**Warnings**
Potential issues PAC detected that may require manual inspection to determine problem severity. Are often non-technical, content-related fixes.

**Comment by lukefretwell** [comment]
I'd add these just to help tab for now. If possible, link to the help page in context.
